<?xml version="1.0" ?>
<launch>
  <!-- <param name="face_cascade_name" value="$(find hog_haar_person_detection)/config/haarcascade_frontalface_alt.xml" /> -->
  <param name="image_topic" value="/iris_depth_camera/camera/rgb/image_raw"/>
  <node pkg="hog_haar_person_detection" type="person_detect.py" name="person_detection" output="screen"/>
</launch>
